docs/source/dataset.rst

```diff
@@ -0,0 +1,42 @@
+===============
+Sample Dataset
+===============
+Public datasets
+~~~~~~~~~~~~~~~~
+Some data to work with have been obtained from `Zenodo <https://zenodo.org>`_. 
+
++-------------------------------------------------+----------+---------+----------+----------+
+|   Title                                         |Download  |  Format | Dimension| Size     |
++=================================================+==========+=========+==========+==========+
+| `Bullseye probe`_                               |`link1`_  | HDF5    | 4D       | 2.1 GB   |             
++-------------------------------------------------+----------+---------+----------+----------+
+| `Circular probe`_                               |`link2`_  |  HDF5   |  4D      | 2.1 GB   |
++-------------------------------------------------+----------+---------+----------+----------+
+| `Electron Bessel beam diffraction pattern`_     |`link3`_  | DM3     | 3D       | 2.6 GB   |
++-------------------------------------------------+----------+---------+----------+----------+
```

Comment:
Hey @sayandip18, [travis build failure](https://travis-ci.org/github/LiberTEM/LiberTEM/jobs/661018080?utm_medium=notification&utm_source=github_status) needs to be fixed.